packages/models/proto/v2/agreement/events.proto

```diff
@@ -74,3 +74,10 @@ message AgreementConsumerDocumentRemovedV2 {
   AgreementV2 agreement = 2;
 }
 
+message AgreementPutInDraftByPlatformV2 {
+  AgreementV2 agreement = 1;
+}
+
+message AgreementPutInMissingCertifiedAttributesByPlatformV2 {
+  AgreementV2 agreement = 1;
+}
```

Comment:
@galales temporary names for the two new events we discussed - before merging, let's agree on the naming :)

Reply:
This is very hard...no better suggestions except
```suggestion
message AgreementSetInDraftByPlatformV2 {
  AgreementV2 agreement = 1;
}

message AgreementSetInMissingCertifiedAttributesByPlatformV2 {
  AgreementV2 agreement = 1;
}
```